src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -109,13 +109,13 @@ namespace
         ParamIdAdaptiveTileMaxSamples                   = 43,
         ParamIdAdaptiveTileNoiseThreshold               = 44,
         ParamIdBackgroundAlphaValue                     = 15,
-        ParamIdNoiseSeed                                = 76,
-        ParamIdEnableNoiseSeed                          = 77,
+        ParamIdPerFrameNoiseSeedVariation               = 76,
```

Comment:
The name for this one was correct before. It's not per-frame noise seed variation but the opposite: a fixed noise seed for all frames.

Reply:
True, copied and pasted it before too fast. I reverted it.